docs/docs/guides/contributing-components.md

```diff
@@ -3,12 +3,6 @@ title: Contributing Components
 sidebar_label: Contributing Components
 ---
 
-:::warning
```

Comment:
Why was this removed? The repo is still empty. I think we should put our own components first, so that they serve as an example.

Reply:
It read a bit confusing and imprecise: Of course, you can already contribute components. It's just that the catalog mechanism is not complete yet. I updated it to reflect that.

Reply:
Ok, feel free to approve, as I cannot do it :smile: